src/k8s/selc_ingress.tf

```diff
@@ -21,50 +57,76 @@ resource "kubernetes_ingress" "selc_ingress" {
             service_name = "hub-spid-login-ms"
             service_port = var.default_service_port
           }
-          path = "/hub-spid-login-ms/(.*)"
+          path = "/spid/v1/(.*)"
         }
 
         path {
           backend {
             service_name = "b4f-dashboard"
             service_port = var.default_service_port
           }
-          path = "/b4f-dashboard/(.*)"
+          path = "/dashboard/v1/(.*)"
         }
 
         path {
           backend {
-            service_name = "pdnd-interop-uservice-party-process"
-            service_port = 8088
+            service_name = "ms-product"
+            service_port = var.default_service_port
           }
-          path = "/pdnd-interop-uservice-party-process/(.*)"
+          path = "/ms-product/(.*)"
         }
 
+      }
+    }
+  }
+}
+
+resource "kubernetes_ingress" "selc_pdnd_ingress" {
+  depends_on = [helm_release.ingress]
+
+  metadata {
+    name      = "${kubernetes_namespace.selc.metadata[0].name}-pdnd-ingress"
+    namespace = kubernetes_namespace.selc.metadata[0].name
+    annotations = {
+      "kubernetes.io/ingress.class"                    = "nginx"
+      "nginx.ingress.kubernetes.io/rewrite-target"     = "/pdnd-interop-uservice-$1/0.1/$2"
```

Comment:
maybe better use `0.1` as terraform var

Reply:
the version 0.1 is statically mapped to the v1 version of the ingress' path
I suggest to let it be a costant and not defined as variable